CONTRIBUTING.md

```diff
@@ -158,10 +160,10 @@ Examples:
     qt: Add feed bump button
     log: Fix typo in log message
 
-The body of the pull request should contain enough description about what the
-patch does together with any justification/reasoning. You should include
-references to any discussions (for example other tickets or mailing list
-discussions).
+The body of the pull request should contain sufficient description of *what* the
+patch does, and even more importantly, *why*, with justification and reasoning.
+You should include references to any discussions (for example, other tickets or
+mailing list discussions). The PR body should not contain any `@` mentions.
```

Comment:
I think "The PR body should not contain any `@` mentions." is a great addition to avoid maintainers or scripts needing to edit things, but I worry it's a bit unclear---"PR body" might be inferred as meaning that no comment on a PR should mention a user.  Also, I recently read some wise advice that "and even more importantly, [describe] *why* [something should be changed], with justification and reasoning."  :-)

I suggest: "The description for a new PR should not contain any `@` mentions.  The PR description will be included in the commit message when the PR is merged and any users mentioned in the description will be annoyingly notified each time a fork of Bitcoin Core copies the merge.  Instead, make any username mentions in a subsequent comment to the PR."